Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -38,8 +38,21 @@ public class ApplicationPOMModel : ApplicationModelBase
     {       
         public ApplicationPOMModel()
         {
+            this.OnDirtyStatusChanged += POM_OnDirtyStatusChanged;            
         }
-
+        private void POM_OnDirtyStatusChanged(object sender, EventArgs e)
```

Comment:
Change its name to OnDirtyStatusChanged

Reply:
after changing OnDirtyStatusChanged method name getting error because it's already the event name so I put OnDirtyStatusChanged1